src/util/config/config.cpp

```diff
@@ -359,19 +359,23 @@ namespace dxvk {
       { "d3d11.dcSingleUseMode",            "False" },
     }} },
     /* Rockstar Games Launcher                    */
-    { R"(\\Rockstar Games\\Launcher\\Launcher\.exe$)", {{
+    { R"(\\Rockstar Games\\Launcher\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* Rockstar Social Club                       */
-    { R"(\\Rockstar Games\\Social Club\\SocialClubHelper\.exe$)", {{
+    { R"(\\Rockstar Games\\SocialClub\\SocialClubHelper\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* EA Desktop App                             */
     { R"(\\EADesktop\.exe$)", {{
       { "dxvk.maxChunkSize",                "1"   },
     }} },
     /* Origin app (legacy EA Desktop)             */
-    { R"(\\Origin\.exe$)", {{
+    { R"(\\Origin\\OriginWebHelperService\.exe$)", {{
+      { "dxvk.maxChunkSize",                "1"   },
+    }} },
+    /* Ubisoft Connect (UPlay)                    */
+    { R"(\\UbisoftConnect\\upc\.exe$)", {{
```

Comment:
If you want to include both then it should be `R"(\\(UbisoftConnect|upc)\.exe$)"`

Reply:
It is because the syntax `\\UbisoftConnect\\upc\.exe$` means that it will only apply the config to `upc.exe` if it resides in a folder called `UbisoftConnect` so i got confused and assumed you had a game install where that was the case.